samples/openthread/cli/README.rst

```diff
@@ -71,7 +71,10 @@ The sample supports the following development kits for testing the network statu
 
 .. table-from-rows:: /includes/sample_board_rows.txt
    :header: heading
-   :rows: nrf52840dk_nrf52840, nrf52833dk_nrf52833
+   :rows: nrf5340dk_nrf5340_cpuapp, nrf52840dk_nrf52840, nrf52833dk_nrf52833
```

Comment:
This will fail in the doc build for now, since nrf5340dk_nrf5340_cpuapp doesn't exist in sample_board_rows.txt yet.
It will be added though, so we need to see which PR is ready first. ;)